go/pkg/cosmos/block.go

```diff
@@ -63,34 +60,22 @@ func (s *BlockService) GetBlock(height int) (*Block, error) {
 }
 
 func (c *HTTPClient) GetBlock(height *int) (*Block, error) {
-	logger.Infof("GetBlock() %s", height)
 	var res *BlockResponse
 	var resErr *struct {
 		RPCErrorResponse
 		Message string `json:"message"`
 	}
-	logger.Info("GetBlock() 1")
 	req := c.tendermint.R().SetResult(&res).SetError(&resErr)
```

Comment:
are we intentionally passing a pointer to a pointer here? 🤔

Reply:
no idea